### ISSUE TITLE
Multiple definitions of `mutex_log`

### ISSUE THREAD

**Issue by tuliom** [libnxz/power-gzip#9]
@lucmaga found out an issue while building libnxz with GCC 10:

```
/usr/lib64/gcc/powerpc64le-suse-linux/10/../../../../powerpc64le-suse-linux/bin/ld: nx_deflate.o:(.bss+0x0): multiple definition of `mutex_log'; nx_inflate.o:(.bss+0x0): first defined here
/usr/lib64/gcc/powerpc64le-suse-linux/10/../../../../powerpc64le-suse-linux/bin/ld: nx_zlib.o:(.bss+0x8228): multiple definition of `mutex_log'; nx_inflate.o:(.bss+0x0): first defined here
/usr/lib64/gcc/powerpc64le-suse-linux/10/../../../../powerpc64le-suse-linux/bin/ld: gzip_vas.o:(.bss+0x18): multiple definition of `mutex_log'; nx_inflate.o:(.bss+0x0): first defined here
/usr/lib64/gcc/powerpc64le-suse-linux/10/../../../../powerpc64le-suse-linux/bin/ld: nx_compress.o:(.bss+0x0): multiple definition of `mutex_log'; nx_inflate.o:(.bss+0x0): first defined here
/usr/lib64/gcc/powerpc64le-suse-linux/10/../../../../powerpc64le-suse-linux/bin/ld: nx_uncompr.o:(.bss+0x0): multiple definition of `mutex_log'; nx_inflate.o:(.bss+0x0): first defined here
collect2: error: ld returned 1 exit status
make: *** [Makefile:33: libnxz.so] Error 1
```

**Comment by tuliom** [comment]
This is a [new behavior in GCC 10](https://gcc.gnu.org/gcc-10/changes.html).

While this issue is not fixed, one can continue to build libnxz with
```
make CC=gcc-10 FLG="-fcommon -std=gnu11"
```

**Comment by rzinsly** [comment]
Done by #21.

**Comment by rzinsly** [comment]
The patch introduced another error, reopening.

**Comment by rzinsly** [comment]
Fixed by be87cbb.
